x/emissions/keeper/keeper.go

```diff
@@ -1251,8 +1251,11 @@ func (k *Keeper) GetReputerAddressByP2PKey(ctx context.Context, p2pKey string) (
 // Returns ((0,0), true) if there was no prior topic weight set, else ((x,y), false) where x,y!=0
 func (k *Keeper) GetPreviousTopicWeight(ctx context.Context, topicId TopicId) (alloraMath.Dec, bool, error) {
 	topicWeight, err := k.previousTopicWeight.Get(ctx, topicId)
-	if errors.Is(err, collections.ErrNotFound) {
-		return alloraMath.ZeroDec(), true, nil
+	if err != nil {
+		if errors.Is(err, collections.ErrNotFound) {
+			return alloraMath.ZeroDec(), true, nil
```

Comment:
Is this used down the road in a division? 
If so then it should not be 0 but epsilon. 
But if that's not the case, 0 is ok I guess.

Reply:
It is not. We are handling potential division by zero cases in the function scope, not in keepers atm